scheduler/crates/api/src/account/remove_account_integration.rs

```diff
@@ -7,14 +7,14 @@ use nettu_scheduler_infra::NettuContext;
 
 pub async fn remove_account_integration_controller(
     http_req: web::HttpRequest,
-    path: web::Json<PathParams>,
+    mut path: web::Json<PathParams>,
```

Comment:
I just noticed that this should be: `mut path: web::Path<PathParams>`. This is an old bug not related to this PR :)